src/Ilios/CoreBundle/Classes/UserMaterial.php

```diff
@@ -39,6 +44,20 @@ class UserMaterial
      * @IS\Expose
      * @IS\Type("string")
      */
+    public $clm;
+
+    /**
+     * @var int
+     * @IS\Expose
+     * @IS\Type("string")
+     */
+    public $slm;
```

Comment:
I think there is some value in expanding this property to be `sessionLearningMaterial` for long term clarity.